fontra2fontir/src/source.rs

```diff
@@ -59,11 +67,16 @@ impl FontraIrSource {
     }
 
     fn load_glyphinfo(&mut self) -> Result<(), Error> {
+        if !self.glyph_info.is_empty() {
```

Comment:
is this expecting to be called multiple times, and we're just trying to avoid doing duplicate work?

Reply:
It's not really supposed to get called over and over but doing so felt achievable enough the check felt worth it. You could argue it shouldn't be there.